src/components/ProfilePage/ProfilePage.jsx

```diff
@@ -250,7 +273,27 @@ export function ProfilePage() {
                 <TalentData {...propsTalentData} ref={talentDataRef} />
                 <About {...propsAbout} ref={aboutRef} />
             </div>
+            <div className={s.proofs_side}>
+                <div className={s.updating_proofs}>
+                    <img
+                        className={`${s.add} ${
+                            isRotated && proofsFormActive ? s.rotated : ""
+                        }`}
+                        onClick={() => {
+                            setproofsFormActive(!proofsFormActive);
```

Comment:
Better to use previous state than write like this
```suggestion
                            setproofsFormActive((prev)=>!prev);
```